packages/deprecation-crawler/src/lib/commands/default.command.ts

```diff
@@ -25,22 +26,22 @@ export const defaultCommand: YargsCommandObject = {
             (config: CrawlConfig): CrawlerProcess =>
               askToSkip('Grouping?', group(config), {
                 precondition: async (r) =>
-                  r.deprecations?.some((d) => !d.group)
+                  r.deprecations?.some((d) => !d.group),
               }),
+            saveDeprecations,
```

Comment:
Added `saveDeprecations` to default command because otherwise the raw deprecations file wouldn't be created if the grouping phase is skipped.